### ISSUE TITLE
Accessibility - Country List Screen - TalkBack Not Announcing Drop-down List

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#90]
TalkBack is not announcing the drop-down list on the "By Level" and "By Region" tabs.

TalkBack reads the first item in the list but does not tell the user that it's a list or they can tab it to get a given level or region. If you tap it, then it shows you a list and says how many lists are in the list.

**Comment by jfc3-dol** [comment]
TalkBack is still not announcing the drop-down list on the "By Level" and "By Region" tabs, so the user knows they have a choice instead of the default option.

TalkBack reads the first item in the list but does not tell the user that it's a list, or they can tab it to get a given level or region. If you tap it, then it shows you a list and says how many items are in the list.

**Comment by harpreetsdev** [comment]
@jfc3-dol 
Please retest this issue as well. Cannot recreate.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

Once I get the build from the OPA Tech team, most likely early next week, I will test this again.

**Comment by jfc3-dol** [comment]
@pbhatt17 

TalkBack now announces the drop-down list and how many items for each of the sectors. So I'm going to close this issue.
